services/ui-src/src/components/cards/TaIntroductionCard.tsx

```diff
@@ -9,19 +9,18 @@ import { ReportIntroCardActions } from "./ReportIntroCardActions";
  */
 export const TaIntroductionCard = () => {
   return (
-    <ReportIntroCard title="HCBS Timely Access Report">
-      The HCBS is ...
-      <Link
-        isExternal={true}
-        href="https://www.govinfo.gov/content/pkg/PLAW-109publ171/pdf/PLAW-109publ171.pdf"
-      >
-        6071(a)(1) of the Deficit Reduction Act (DRA)
-      </Link>
-      as "increasing the use of home and community-based, rather than
-      institutional, long-term care services."
+    <ReportIntroCard title="Timely Access Compliance Measure Report (TACM) Report">
```

Comment:
Ticket says: "Timely Access Compliance Measure Report (TACM)" but then the Figma shows 
"Timely Access Compliance Measure (TACM) Report"
Can you confirm with Cameron and Megan which one they prefer instead of 
"Timely Access Compliance Measure Report (TACM) Report"

Reply:
Checked with Cam and whats correct and fixed